varnish-mixin/README.md

```diff
@@ -0,0 +1,114 @@
+# Varnish Cache mixin
+
+Varnish Cache mixin contains a configurable dashboard and set of alerts.
+
+The Varnish Cache mixin contains the following dashboards:
+
+- Varnish overview
+
+and the following alerts:
+
+- VarnishCacheLowCacheHitRate
+- VarnishCacheHighMemoryUsage
+- VarnishCacheHighCacheEvictionRate
+- VarnishCacheHighSaturation
+- VarnishCacheSessionsDropping
+- VarnishCacheBackendFailure
+- VarnishCacheBackendUnhealthy
+
+## Varnish Cache overview
+
+The Varnish overview dashboard delivers a breakdown of metrics such as thread utilization, client and backend requests + connections, cache, network throughput, and client and backend log data, offering an in-depth perspective on connections and communication. [Promtail and Loki needs to be installed](https://grafana.com/docs/loki/latest/installation/) and provisioned for logs with your Grafana instance.
+
```

Comment:
Only thing I would add here is dashboard screenshots. 
Here's an example from SAP HANA:
```
![First screenshot of the SAP HANA instance overview dashboard](https://storage.googleapis.com/grafanalabs-integration-assets/sap-hana/screenshots/sap-hana-instance-overview-1.png)
```
They should go in the right spot if you change the sap-hana spots to your varnish ones, and just name the screenshots the same as the file you put here.